docs/applications/create-commit-statuses-from-application-deployment-statuses.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Create Commit Statuses from Application Deployment Statuses
+intro: Outgoing Webhooks allow users to integrate Devopness to their current CI/CD Workflows. Learn how to create outgoing webhooks to update a commit status check to match your Devopness application deploy pipeline status.
```

Comment:
@thlmenezes following the `agnostic` approach, use `commit status` isn't a good practice, because others source providers may not follow this pattern, like `Azure Devops`

https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pull-request-statuses?view=azure-devops-rest-7.0

Reply:
@vianaz I agree with you that the text can be improved to be more source provider agnostic. I'll work on it.

P.S.: I believe the following API to create a commit status on Azure can be a better parallel to the post, but I see your point for the future for this documentation

https://learn.microsoft.com/en-us/rest/api/azure/devops/git/statuses/create?view=azure-devops-rest-7.0&tabs=HTTP

Reply:
Intro updated